Packs/CommonPlaybooks/Playbooks/playbook-Search_And_Delete_Emails_-_Generic_-_v2_6_1.yml

```diff
@@ -39,19 +35,20 @@ tasks:
     quietmode: 0
     isoversize: false
     isautoswitchedtoquietmode: false
+    continueonerrortype: ""
   "1":
     id: "1"
     taskid: 0a7ca589-107b-48e4-8705-2becca697165
     type: playbook
     task:
       id: 0a7ca589-107b-48e4-8705-2becca697165
       version: -1
-      name: Search And Delete Emails - EWS
+      name: search_and_delete_emails_-_ews
       description: This playbook searches EWS to identify and delete emails with similar attributes of a malicious email.
-      playbookName: Search And Delete Emails - EWS
       type: playbook
       iscommand: false
       brand: ""
+      playbookId: search_and_delete_emails_-_ews
```

Comment:
The playbook id here was also changed - maybe it's due to custom content you played with on your env?